src/IntegrationTests/Applications/SampleAspNetCoreApp/Controllers/WeatherForecastController.cs

```diff
@@ -3,14 +3,23 @@
 using System.Linq;
 using System.Net.Http;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Configuration;
+using NewRelic.Telemetry.Metrics;
 
 namespace SampleAspNetCoreApp.Controllers
 {
     [ApiController]
     [Route("[controller]")]
     public class WeatherForecastController : ControllerBase
     {
+        private readonly IConfiguration _config;
+
+        public WeatherForecastController(IConfiguration config) 
```

Comment:
Let's use dependency injection to build the metric data sender and send it in.
Can we do the same for the Opentelemetry stuff if it is not already like this.